pkg/client/observatorium/observability_config.go

```diff
@@ -80,6 +70,74 @@ type ObservabilityCloudWatchLoggingConfig struct {
 	configFilePath                string                                                      `validate:"-"`
 }
 
+type DataPlaneObservabilityOIDCCredentials struct {
+	ClientID     string `yaml:"client_id" validate:"required"`
+	ClientSecret string `yaml:"client_secret" validate:"required"`
+}
+
+type DataPlaneObservabilityOIDCConfiguration struct {
+	AuthorizationServer string                                 `yaml:"authorization_server" validate:"required"`
+	Realm               string                                 `yaml:"realm" validate:"required"`
+	Credentials         *DataPlaneObservabilityOIDCCredentials `yaml:"credentials" validate:"required,dive"`
+}
+
+type DataPlaneObservabilityRemoteWriteConfiguration struct {
+	RemoteWriteUrl    string                                   `yaml:"remote_write_url" validate:"required"`
```

Comment:
Well known acronyms should be written in capitalized letters as long as it does not affect exportability of the variables / types. In this case, it should be called `RemoteWriteURL